opentelemetry-api/src/opentelemetry/baggage/propagation/__init__.py

```diff
@@ -11,15 +11,24 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
-#
-import typing
+
 import urllib.parse
+from logging import getLogger
+from re import compile as compile_
+from typing import Iterable, Mapping, Optional, Set
 
 from opentelemetry import baggage
 from opentelemetry.context import get_current
 from opentelemetry.context.context import Context
 from opentelemetry.propagators import textmap
 
+_logger = getLogger(__name__)
```

Comment:
can we document a link to the spec doc that specifies these rules?

Reply:
+1, and did you find this regex verbatim somewhere, or should we review it for correctness 😅